mage_integrations/mage_integrations/sources/stripe/__init__.py

```diff
@@ -952,21 +965,22 @@ def recursive_to_dict(some_obj):
 
 
 def sync_event_updates(stream_name, is_sub_stream):
-    """
+    '''
```

Comment:
Why do you change it to three single quote here?

Reply:
Modified on 028336556fca308242837d69d26186c17481362d